minimal/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "build": "react-router build",
+    "dev": "react-router dev",
+    "start": "react-router-serve ./build/server/index.js",
+    "typecheck": "react-router typegen && tsc"
+  },
+  "dependencies": {
+    "@react-router/node": "*",
+    "@react-router/serve": "*",
+    "isbot": "^5.1.17",
+    "react": "^19.0.0",
+    "react-dom": "^19.0.0",
+    "react-router": "*"
+  },
+  "devDependencies": {
+    "@react-router/dev": "*",
+    "@types/node": "^20",
+    "@types/react": "^19.0.1",
+    "@types/react-dom": "^19.0.1",
+    "tailwindcss": "^3.4.17",
```

Comment:
At the moment tailwindcss v4 with the Vite plugin [doesn't work with StackBlitz](https://github.com/tailwindlabs/tailwindcss/issues/13133), which is what this template is primarily for